SlugifierInterface.php

```diff
@@ -9,21 +9,31 @@
  * file that was distributed with this source code.
  */
 
+namespace Symfony\Cmf\Bundle\CoreBundle\Slugifier;
+
+if (!interface_exists('Symfony\Cmf\Bundle\CoreBundle\Slugifier\SlugifierInterface')) {
+    /**
+     * @deprecated Since 1.0, to be removed in 2.0.
+     */
+    interface SlugifierInterface {
```

Comment:
I prefer this method over that one. Your method means that a CoreBundle namespaced interface is suddently autoloaded (and included in the dumped classmap) when using this package, whereas this solution completely hides the interface from the outside world (it's only defined when using this interface).